_specifications/lsp/3.18/language/completion.md

```diff
@@ -641,11 +640,10 @@ export interface CompletionItem {
 
 	/**
 	 * An edit which is applied to a document when selecting this completion.
-	 * When an edit is provided the value of `insertText` is ignored.
+	 * When an edit is provided, the value of `insertText` is ignored.
 	 *
 	 * *Note:* The range of the edit must be a single line range and it must
-	 * contain the position at which completion has been requested. Despite this
-	 * limitation, your edit can write multiple lines.
+	 * contain the position at which completion has been requested.
```

Comment:
Why did you remove 

> Despite this limitation, your edit can write multiple lines.

I added this since clients weren't aware that this is possible

Reply:
I probably thought that part was redundant as it's implicitly true given the definition of ranges, but if there already was confusion about this from clients, I'll add it back—my bad!